junit-resource-poller/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
why do we need this?

Reply:
I guess to allow mocking of the final class `HttpPollingResource`

Reply:
Yep, `HttpPollingResource` is unfortunately final.

Alternative is to relax `FailureCachingHttpPollingResource` to take in any `ExternalResource` as a delegate, but I'm wary of that; we're pretty tightly coupled to what `HttpPollingResource` does (noop `after()`,  skippable `before()`)